playbooks/generate-csr.yaml

```diff
@@ -0,0 +1,34 @@
+- hosts: nerc-bootstrap
+  gather_facts: false
+  vars:
+    common_names:
+      - api.nerc-ocp-prod.rc.fas.harvard.edu
+      - apps.nerc-ocp-prod.rc.fas.harvard.edu
+    country_name: US
+    organization_name: NERC
+    state_or_province_name: MA
+    dirs:
+      - private
+      - csr
+
+  tasks:
+    - name: create directories if missing
+      file:
+        path: "/etc/ssl/{{ item }}"
+        state: directory
+      loop: "{{ dirs }}"
```

Comment:
Is this installing things into `/etc` on `nerc-bootstrap` because they're operating a CA there? Or could this just run on `localhost` and create the CSR locally?

Reply:
My intention here was to enable future automation, where you would be able to generate then pickup the csrs on nerc-bootstrap and then drop the CA signed certs there, eventually being able to deploy them from nerc-bootstrap.